app/views/past_foreign_secretaries/index.html.erb

```diff
@@ -172,612 +172,610 @@
 </div>
 
 <div class="govuk-grid-row featured-profiles foreign-secretaries">
-  <header class="govuk-grid-column-one-quarter govuk-!-padding-0 govuk-!-padding-top-3 heading">
+  <header class="govuk-grid-column-full govuk-!-padding-left-0 govuk-!-padding-bottom-3 heading">
```

Comment:
I've just realised (although this probably out of scope) that having a header element here is potentially unnecessary - it's not in a section or anything, is it? I'm not sure it adds anything.

Reply:
Also done.